scripts/run_travis_tests.sh

```diff
@@ -22,7 +22,7 @@ sparkVersion=2.4.4
 hadoopVersion=2.7
 
 if [ "$mode" = "unit" ]; then
-  mvn clean install -Punit-tests -DskipTests -q
+  mvn clean install -DskipTests -q
```

Comment:
@xushiyan why did this have to change? you are probably right. just trying to understand

Reply:
sure. the "unit-tests" maven profile sets `skipUTs=false`, which will cause scalatest-maven-plugin to run tests during the install phase. I actually don't recall why I have to set maven profile here.. 🤔

Reply:
got it.